kore/src/Kore/Attribute/SourceLocation.hs

```diff
@@ -52,6 +53,8 @@ data SourceLocation = SourceLocation
 instance Default SourceLocation where
     def = SourceLocation def def
 
+notDefault :: SourceLocation -> Bool
+notDefault = (/=) def
```

Comment:
Where is this used?

Reply:
Here https://github.com/kframework/kore/pull/2594/files#diff-10bb26c5f6fb3d7794de9cbf4fc75e448ce311e8673315c27ffb7b0ef7f98a9eR497